csharp/build-native.sh

```diff
@@ -0,0 +1,20 @@
+
+#!/bin/sh
+if buck --version >/dev/null 2>&1; then true; else
+echo "Building Buck!"
+mkdir lib
+cd lib
+git clone https://github.com/facebook/buck.git
+cd buck
+ant
+cd ..
+cd ..
+fi
+#!/bin/bash
+
+
+buck build //:yoga
```

Comment:
we don't need this line since this rule will be built automatically by following rules